### DISCUSSION TITLE
Mesh reduce number of vertices

### DISCUSSION THREAD

**Discussion by gnicolas73** [brainvisa/brainvisa.github.io#63] [Q&A] [answered]
Hello everyone, 

I use the Morphologist pipeline to create the cortical surface from an MRI, I obtained the mesh of the left and the right hemispheres and then I concatenate the both hemisphere meshes to obtain only one mesh for the entire brain.

However, the number of vertices in each hemisphere is equal to 67,032 and for the entire brain it is equal to 134,867. That is a lot and I want to reduce this number in order to have about 10,000 vertices in the entire brain. How can I do that ? 

Thank your for your help, 
Gaelle

**Comment by denisri** [comment, marked as answer]
Hi,
You can try using the command AimsMeshDecimation (use -h to get help) with more "aggressive" parameters than the ones used in Morphologist. But 10000 vertices for the whole brain is not much, you will lose many details...
Denis
